src/config/config_test.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+)
+
+func TestConfig_Keys(t *testing.T) {
+	var nodeConfig NodeConfig
+
+	if nodeConfig.PublicKey != "" {
```

Comment:
I don’t think these assertions can ever fail when doing `var nodeConfig NodeConfig`, since the values should be their empty defaults.